### ISSUE TITLE
Game crash

### ISSUE THREAD

**Issue by Tim020** [Tim020/Tiny-Storage#29]
Game crashes when trying to break a modular chest by hand in normal gamemode.
Crash log: https://gist.github.com/Tim020/5c0ab65753119f553c69e4ab7699a84f

**Comment by Tim020** [comment]
@OrionDevelopment you seen this?

**Comment by marchermans** [comment]
Yes i have, no i have not checked it out. Probably something with the blockstate that is not really working correctly.

**Comment by marchermans** [comment]
We need to wait for https://github.com/MinecraftForge/MinecraftForge/pull/3086 to be accepted. It is missing data.

**Comment by marchermans** [comment]
Also now waiting for https://github.com/MinecraftForge/MinecraftForge/pull/3132
